modules/swagger-codegen/src/main/java/io/swagger/codegen/InlineModelResolver.java

```diff
@@ -191,6 +191,12 @@ public void flatten(Swagger swagger) {
         }
     }
 
+    private void fixStringModel(ModelImpl m) {
+        if (m.getType() != null && m.getType().equals("string") && m.getExample() != null) {
+            m.setExample(m.getExample().toString().replace("\"", ""));
```

Comment:
Yes you are right, I didn't think about this possible scenario. I will submit something today to fix this. Thanks for the review @wing328 and @ePaul